domains.txt

```diff
@@ -581,26 +642,33 @@ rightfutures.com
 rjcgroup.co.uk
 rmirecruitment.com
 rmw-uk.com
+roberthalf.co.uk
 robertwalters.com
 roc-search.com
+rothsteins.co.uk
 rpint.com
 rtrs.co.uk
 rubeus-solutions.com
+rubiconconsulting.co.uk
 rullion.co.uk
 s-sa.co.uk
 sabre-rm.com
 salesjobsoffshore.co.uk
 sandersonplc-update.com
 sandersonplc.com
+saragossa.co.uk
 satigo.com
+sbhassociates.co.uk
 sceneone.uk.com
 scopeat.co.uk
+scott-merrick.co.uk
```

Comment:
Our philosophy is straightforward. We recognise that business is essentially to do with people; we understand the necessity of building strong relationships with the people we represent.

Successful recruiting relies on integrity, trust and the ability to listen. At Scott Merrick we value and work by those principles, offering clients and candidates alike, an individualised and professional service, and one that is geared for the long term.